odml/property.py

```diff
@@ -96,7 +97,23 @@ def dtype(self, new_type):
     @property
     def parent(self):
         """the section containing this property"""
-        return self._section
+        return self._parent
+
+    @parent.setter
+    def parent(self, new_parent):
+        if new_parent is None:
+            return
```

Comment:
not sure: If I actually want to make the property stand-alone, I need to be able to set the parent to None...

I would vote for accepting None here and setting it.